src/runtime/CMakeLists.txt

```diff
@@ -145,11 +145,27 @@ set(RUNTIME_HEADER_FILES
 # by add_custom_command, as is the case in this directory.
 add_library(Halide_initmod OBJECT)
 
-set(CXX_WARNING_FLAGS -Wall -Werror -Wno-unused-function -Wcast-qual)
-set(RUNTIME_CXX_FLAGS -O3
-    -fno-vectorize -ffreestanding -fno-blocks -fno-exceptions -fno-unwind-tables
+# Note: ensure that these flags match the flags in the Makefile.
+# Note: this always uses Clang-from-LLVM for compilation, so none of these flags should need conditionalization.
+set(RUNTIME_CXX_FLAGS
+    -O3
+    -ffreestanding
```

Comment:
This is more than warnings - a lot of these are really important (mostly freestanding)

Reply:
The non-warnings were set before. I just reformatted to be one-per-line and alphabetical order.

Reply:
oops, right you are. I was wondering why anything was working.